content/news/028/index.md

```diff
@@ -610,6 +610,26 @@ This month changes by [@PsichiX]:
 
 [NavMesh]: https://github.com/PsichiX/navmesh
 
+### [Rust CUDA]
+
+[Rust CUDA] by [Riccardo D'Ambrosio] is a newly-released project with the goal
+of making Rust a Tier-1 language for fast GPU computing. There are still many
+bugs, and it's in an early stage.
+
+With this release comes a few crates. `rustc_codegen_nvvm` for compiling Rust to
```

Comment:
> No bold/italic/etc formatting <...>

Maybe it's better to convert these to links?

Reply:
Ah, that does make sense. This is just a habit I have from the Veloren blogs :P will fix this!